src/test/java/core/basesyntax/dao/ReportWorkerTest.java

```diff
@@ -0,0 +1,90 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.Procedure;
+import core.basesyntax.service.ReportWorker;
+import core.basesyntax.service.fileworker.InputOutputReport;
+import core.basesyntax.strategy.Balance;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.Purchase;
+import core.basesyntax.strategy.Return;
+import core.basesyntax.strategy.Supply;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReportWorkerTest {
+    Map<Procedure, OperationStrategy> operationStrategyMap = new HashMap<>();
+    WarehouseImpl warehouseTest = new WarehouseImpl();
+    InputOutputReport inputOutputReport = new InputOutputReport();
```

Comment:
access modifiers and let's initialize them in @BeforeClass method

Reply:
done